### PR TITLE
 translatation(hs): translate imitation_learning_en

### PR DESCRIPTION
**Pull request by norman26625**

### CODE REVIEW

File: source/02_algo/imitation_learning_en.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
+Imitation learning
```
Comment (VaninaY) [inline source/02_algo/imitation_learning_en.rst:1]: Imitation Learning 
#Capital L

Comment (norman26625) [reply source/02_algo/imitation_learning_en.rst]: done，thx

File: source/02_algo/imitation_learning_en.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+There are still many challenges in current imitation learning, mainly including the following:
+
+- The current imitation learning is for a specific task, and there is a lack of imitation learning methods that can be applied to multiple tasks;
+
+- The current imitation learning algorithm is not optimal for expert data, and it is difficult to surpass expert data to achieve optimal results;
+
+- The current imitation learning algorithm is mainly aimed at observation, and cannot combine multi-modal factors such as speech and natural language;
+
+- Current imitation learning can find local optima, but often cannot find global optima.
```
Comment (puyuan1996) [inline source/02_algo/imitation_learning_en.rst:54]: There are still many challenges in imitation learning, mainly including the following:

- Generally speaking, it is for a specific task, and there is a lack of imitation learning methods that can be applied to multiple tasks;

- For situations where the data is not optimal, it is difficult to surpass the data to achieve optimal results;

- Mainly focus on the research of observation, without combining multi-modal factors such as speech and natural language;

- The local optimum can be found, but the global optimum can often not be found.

File: source/02_algo/imitation_learning_en.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+Adversarial Structured IL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The main goal of the Adversarial Structured IL approach is to address the efficiency of IRL. It can be seen from the algorithm of IRL that even if it learns a very good reward function, since the final strategy still needs to perform reinforcement learning steps, if the strategy can be learned directly from expert data, the efficiency can be greatly improved. Based on this idea GAIL
+[8] combines generative network GAN and maximum entropy IRL, which can be continuously and efficiently trained without the need to manually label expert data continuously.
```
Comment (puyuan1996) [inline source/02_algo/imitation_learning_en.rst:38]: The main goal of the Adversarial Structured IL approach is to improve the efficiency of IRL. Even if the IRL algorithm learns a very good reward function, in order to get the final near-optimal policy, it still needs to perform a reinforcement learning step. If the policy can be learned directly from the expert data, the efficiency can be greatly improved. Based on this idea, GAIL [8] combines generative network (GAN) and maximum entropy IRL to learn approximate optimal policies without human annotated expert data.

File: source/02_algo/imitation_learning_en.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+On this basis, many works have improved GAIL. eg InfoGail[9] replaced GAN with WGAN and achieved better results. There are also some recent works such as GoalGAIL [10], TRGAIL [11] and DGAIL [12] which combine other methods such as post-hoc relabeling and DDPG to achieve faster convergence and better final performance.
```
Comment (puyuan1996) [inline source/02_algo/imitation_learning_en.rst:40]: On this basis, many works have made related improvements to GAIL. For example, InfoGail [9] replaced GAN with WGAN and achieved better performance. There are also some recent works such as GoalGAIL [10], TRGAIL [11] and DGAIL [12] which combine other methods such as post-hoc relabeling and DDPG to achieve faster convergence and better final performance.

File: source/02_algo/imitation_learning_en.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+The current research hotspots of BC algorithms mainly focus on two aspects: meta-learning and behavior cloning using VR devices.
+
+
+IRL
+~~~~~~~~
+
+The main goal of IRL is to solve the problem of finding enough high-quality data during data collection. Specifically, IRL first learns a reward function from expert data, and then uses this reward function for subsequent RL training. With such an approach, IRL can theoretically outperform expert data.
```
Comment (puyuan1996) [inline source/02_algo/imitation_learning_en.rst:29]: Inverse reinforcement learning (IRL) is the problem of inferring the reward
function of an agent, given its policy or observed behavior. Specifically, IRL first learns a reward function from expert data, and then uses this reward function for subsequent RL training. IRL can theoretically outperform expert data.

File: source/02_algo/imitation_learning_en.rst [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+From the specific work above, Ziebart et al. [4] first proposed the maximum entropy IRL, which utilizes the maximum entropy distribution to obtain good prospects and effective optimization. Later in 2016, Finn et al. [5] proposed a model-based approach to IRL called Guided Cost Learning (guided cost learning), this method uses a neural network to represent the cost to improve the expressive ability. Subsequently, Hester et al. proposed DQfD [6], which requires only a small amount of expert data, and significantly speeds up training through the pre-training startup process and subsequent learning process. Later methods such as T-REX [7] proposed a structure based on ranking expert data, which indirectly learned the reward function by comparing which expert data performed better.
```
Comment (puyuan1996) [inline source/02_algo/imitation_learning_en.rst:31]: [4] first proposed maximum entropy IRL, which utilizes the maximum entropy distribution to better characterize multimodal behavior for more efficient optimization. In 2016, Finn et al. [5] proposed a model-based approach to IRL called guided cost learning, capable of learning arbitrary nonlinear cost functions, such as neural
networks, without meticulous feature engineering, and formulate an efficient sample-based approximation for MaxEnt IOC. Subsequently, Hester et al. proposed DQfD [6], which requires only a small amount of expert data, and significantly accelerates the training process through pre-training and a specially designed loss function. T-REX [7] propose a novel reward-learning-from-observation algorithm, that extrapolates beyond a set of (approximately) ranked demonstrations in order to infer high-quality reward functions from a set of potentially poor demonstrations.

**Review by PaParaZz1 — APPROVED** [review]
